hack/ds.yaml

```diff
@@ -112,6 +112,12 @@ spec:
     path: gs://ds-operator-engineering-devops/ds-backup-test
     secretName: cloud-storage-credentials
 
+  #### Server Config ####
+  ## Uncomment to set the default server configuration properties for replication
+  # serverConfig:
+  #   groupID: default
+  #   advertisedListenAddress: ""
+
   #### Multi-cluster ####
   # multiCluster:
     # clusterTopology and clusterIdentifier can be used for multiple multi-cluster solutions specifically the
```

Comment:
thes are coming later? what are the expected values in multiCluster

Reply:
No, you just uncomment the section to enable. Its documented in the readme https://github.com/ForgeRock/ds-operator#multi-cluster.
The values will depend on your regions but it defaults to us and eu.

Reply:
I've made the commented section a bit clearer to separate the values and the comments.